get.go

```diff
@@ -172,33 +172,39 @@ func (j *JWKs) refresh() (err error) {
 		return err
 	}
 
-	// Get the JWKs as JSON from the given URL.
+	// Get the JWKS as JSON from the given URL.
 	var resp *http.Response
 	if resp, err = j.client.Do(req); err != nil {
 		return err
 	}
 	defer resp.Body.Close() // Ignore any error.
 
-	// Read the raw JWKs from the body of the response.
+	// Read the raw JWKS from the body of the response.
 	var jwksBytes []byte
 	if jwksBytes, err = ioutil.ReadAll(resp.Body); err != nil {
 		return err
 	}
 
-	// Create an updated JWKs.
-	var updated *JWKs
+	// Only reprocess if the JWKS has changed.
+	if len(jwksBytes) != 0 && bytes.Equal(jwksBytes, j.raw) {
+		return nil
+	}
+	j.raw = jwksBytes
```

Comment:
Shouldn't `j.raw` access be guarded with mutex?

Reply:
An additional mutex is not required because of this existing mutex: https://github.com/MicahParks/keyfunc/blob/1c9ade36ad129fba923b141f26bd59137c2ffff2/get.go#L66